lib/makeFormatter.js

```diff
@@ -1,506 +1,599 @@
-          '_startOfYear.set' + (utc ? 'UTC' : '') + 'Month(0);\n' +
-          '_startOfYear.set' + (utc ? 'UTC' : '') + 'Date(1);\n' +
-          'var _doy = Math.round((d - _startOfYear) / 864e5) + 1;\n';
+      'var _startOfYear = new Date(d.valueOf());\n' +
+      '_startOfYear.set' +
+      (utc ? 'UTC' : '') +
+      'Month(0);\n' +
+      '_startOfYear.set' +
+      (utc ? 'UTC' : '') +
+      'Date(1);\n' +
+      'var _doy = Math.round((d - _startOfYear) / 864e5) + 1;\n';
   }
 
-  if(gets.day){
+  if (gets.day) {
     fnBody += 'var _day = d.get' + (utc ? 'UTC' : '') + 'Day();\n';
   }
 
-  if(gets.year){
+  if (gets.year) {
     fnBody += 'var _year = d.get' + (utc ? 'UTC' : '') + 'FullYear();\n';
   }
 
-  if(gets.hour){
+  if (gets.hour) {
     fnBody += 'var _hour = d.get' + (utc ? 'UTC' : '') + 'Hours();\n';
   }
 
-  if(gets.minutes){
+  if (gets.minutes) {
     fnBody += 'var _mins = d.get' + (utc ? 'UTC' : '') + 'Minutes();\n';
   }
 
-  if(gets.seconds){
+  if (gets.seconds) {
     fnBody += 'var _secs = d.get' + (utc ? 'UTC' : '') + 'Seconds();\n';
   }
 
-  if(gets.millis){
+  if (gets.millis) {
     fnBody += 'var _ms = d.get' + (utc ? 'UTC' : '') + 'Milliseconds();\n';
   }
 
-  if(gets.offset){
+  if (gets.offset) {
     if (utc) {
-      fnBody += 'var _offs = 0, _absOffs = 0, _offH = 0, _offM = 0;'
+      fnBody += 'var _offs = 0, _absOffs = 0, _offH = 0, _offM = 0;';
     } else {
       fnBody +=
-            'var _offs = -d.getTimezoneOffset();\n' +
-            'var _absOffs = _offs < 0 ? -_offs : _offs;\n' +
-            'var _offH = Math.floor(_absOffs / 60);\n' +
-            'var _offM = _absOffs % 60;\n';
+        'var _offs = -d.getTimezoneOffset();\n' +
+        'var _absOffs = _offs < 0 ? -_offs : _offs;\n' +
+        'var _offH = Math.floor(_absOffs / 60);\n' +
+        'var _offM = _absOffs % 60;\n';
     }
   }
 
-  if(gets.week){
-    fnBody +=  // mostly taken from moment, probably not as efficient as it could be
-          'var _wend = ' + (localeWeek.doy - localeWeek.dow) + ';\n' +
-          'var _ddw = ' + (+localeWeek.doy) + ' - d.get' + (utc ? 'UTC' : '') + 'Day();\n' +
-          'if(_ddw > _wend) _ddw -= 7;\n' +
-          'if(_ddw < _wend - 7) _ddw += 7;\n' +
-          'var _d2 = new Date(d.valueOf());\n' +
-          '_d2.set' + (utc ? 'UTC' : '') + 'Date(d.get' + (utc ? 'UTC' : '') + 'Date() + _ddw);\n' +
-          'var _soy2 = new Date(_d2.valueOf());\n' +
-          '_soy2.set' + (utc ? 'UTC' : '') + 'Month(0);\n' +
-          '_soy2.set' + (utc ? 'UTC' : '') + 'Date(1);\n' +
-          'var _doy2 = Math.round((_d2 - _soy2) / 864e5) + 1;\n' +
-          'var _week = Math.ceil(_doy2 / 7);\n' +
-          'var _weekYear = _d2.get' + (utc ? 'UTC' : '') + 'FullYear();\n';
+  if (gets.week) {
+    fnBody += // mostly taken from moment, probably not as efficient as it could be
+      'var _wend = ' +
+      (localeWeek.doy - localeWeek.dow) +
+      ';\n' +
```

Comment:
Similar comment to above (and the same goes for a lot of the ones further down) - previously all the generated code was kept together on the same line where it made sense, but Prettier has split it into many separate lines. String interpolation should make sense here too.